### PR TITLE
Add content/pt-br/TLS(Transport Layer Security).md

### PR DESCRIPTION
**Pull request by MrErlison**
Add 
- content/pt-br/TLS(Transport Layer Security).md

### CODE REVIEW

File: content/pt-br/TLS(Transport Layer Security).md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: TLS
+status: Completed
+category: Conceito
+---
+
+## O que é isso
+
+Transport Layer Security (TLS) é um protocolo projetado para fornecer maior segurança à comunicação através de uma rede. Garante a entrega segura de dados enviados pela Internet, evitando possíveis monitoramentos e/ou alterações dos dados. Este protocolo é amplamente utilizado em aplicativos como mensagens, e-mail, etc.
```
Comment (jessicalins) [inline content/pt-br/TLS(Transport Layer Security).md:9]:
```suggestion
Transport Layer Security (TLS) é um protocolo projetado para fornecer maior segurança à comunicação através de uma rede. Garante a entrega segura de dados enviados pela Internet, evitando possíveis monitoramentos e/ou alterações dos dados. Este protocolo é amplamente utilizado em aplicações como mensagens, e-mail, etc.

```

File: content/pt-br/TLS(Transport Layer Security).md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+category: Conceito
+---
+
+## O que é isso
+
+Transport Layer Security (TLS) é um protocolo projetado para fornecer maior segurança à comunicação através de uma rede. Garante a entrega segura de dados enviados pela Internet, evitando possíveis monitoramentos e/ou alterações dos dados. Este protocolo é amplamente utilizado em aplicativos como mensagens, e-mail, etc.
+
+## Problema relacionado 
+
+Sem o TLS, informações confidenciais, como hábitos de navegação, correspondência por e-mail, bate-papos on-line e chamadas de conferência, podem ser facilmente rastreadas e modificadas por outras pessoas durante a transmissão. Permitir que os aplicativos de servidor e cliente suportem TLS garante que os dados transmitidos entre eles sejam criptografados e não possam ser visualizados por terceiros.
```
Comment (jessicalins) [inline content/pt-br/TLS(Transport Layer Security).md:13]:
```suggestion
Sem o TLS, informações confidenciais, como hábitos de navegação, correspondência por e-mail, bate-papos on-line e chamadas de conferência, podem ser facilmente rastreadas e modificadas por outras pessoas durante a transmissão. Permitir que as aplicações de servidor e cliente suportem TLS garante que os dados transmitidos entre elas sejam criptografados e não possam ser visualizados por terceiros.

```

File: content/pt-br/TLS(Transport Layer Security).md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## Como isso ajuda
+
+O TLS usa uma combinação de técnicas de codificação que fornecem segurança ao transmitir dados através de uma rede. O TLS permite uma conexão criptografada entre um aplicativo cliente e um servidor, tal como um navegador da web e um site bancário. Ele também permite que os aplicativos clientes identifiquem positivamente o servidor que está conectado, o que reduz o risco de um cliente falar com um site fraudulento. Isso garante que terceiros não possam ver e monitorar dados transmitidos entre aplicativos usando o TLS, o que protege informações confidenciais e privadas, como números de cartão de crédito, senhas, localização, etc.
```
Comment (jessicalins) [inline content/pt-br/TLS(Transport Layer Security).md:17]:
```suggestion
O TLS usa uma combinação de técnicas de codificação que fornecem segurança ao transmitir dados através de uma rede. O TLS permite uma conexão criptografada entre uma aplicação cliente e um servidor, tal como um navegador da web e um site bancário. Ele também permite que as aplicações clientes identifiquem positivamente o servidor que está conectado, o que reduz o risco de um cliente se comunicar com um site fraudulento. Isso garante que terceiros não possam ver e monitorar dados transmitidos entre aplicações usando o TLS, o que protege informações confidenciais e privadas, como números de cartão de crédito, senhas, localização, etc.

```

**Review by jessicalins — COMMENTED** [review]
looking good, just a few suggestions, mainly the change from aplicativos -> aplicações

File: content/pt-br/TLS(Transport Layer Security).md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: TLS
+status: Completed
+category: Conceito
```
Comment (jessicalins) [inline content/pt-br/TLS(Transport Layer Security).md:4]:
```suggestion
category: conceito
```

**Review by edsoncelio — APPROVED** [review]

**Pull request merged**